web_content.cfg

```diff
@@ -37,20 +37,23 @@
         tag = pre
       </filter>
     </content>
-    <image five_min>
-      tries = 3
-      sleep = 10
-      file = ACE_5min.gif
-      warn_bad_image = 1
-      <filter>
-        tag = img
-        src = mta_ace_plot\.gif
-      </filter>
-    </image>
+    # Fetched with get_swpc.py instead
+    #<image five_min>
+    #  tries = 3
+    #  sleep = 10
+    #  file = ACE_5min.gif
+    #  warn_bad_image = 1
+    #  <filter>
+    #    tag = img
+    #    src = mta_ace_plot\.gif
+    #  </filter>
+    #</image>
   </ace>
 
   <space_weather>
-    url = http://services.swpc.noaa.gov/text/sgarf.txt
+    # sgarf.txt is fetched with get_swpc.py
+    #url = https://services.swpc.noaa.gov/text/sgarf.txt
+    url = file:///proj/sot/ska/ska/data/arc/sgarf.txt
```

Comment:
This won't work in a dev ska but I couldn't remember what any of this config-templated file-parsing was doing so I just tried to hack in the independently updated text file.